backend-tests/tests/test_deployments.py

```diff
@@ -552,7 +552,7 @@ def compare_phases_json(expected, response):
                 assert exp[k] == rsp[k]
 
 
-def setup_devices_and_management_st(nr_devices=100):
+def setup_devices_and_management_st(nr_devices=100, deploy_to_group=""):
```

Comment:
Suggestion:
I'm not a pythonista, but I do think it is common to:
```suggestion
def setup_devices_and_management_st(nr_devices=100, deploy_to_group=None):
```

And then the test can be:
```
if deploy_to_group:
    ...
```

Reply:
sure! looks prettier:) I just pushed that and since that is cosmetic I will not re-run the pipelines.